pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -591,10 +590,11 @@ func verifySignature(connSignature *ConnectionSignature, recipientKeys string) (
 	pubKey := base58.Decode(recipientKeys)
 
 	// TODO: Replace with signed attachments issue-626
-	signatureSuite := ed25519signature2018.New()
+	suiteVerifier := &ed25519signature2018.PublicKeyVerifier{}
+	signatureSuite := ed25519signature2018.New(ed25519signature2018.WithVerifier(suiteVerifier))
 
 	err = signatureSuite.Verify(&verifier.PublicKey{
-		Type:  kms.Ed25519Type,
+		Type:  "ED25519",
```

Comment:
why this change?

Reply:
I commented that we tie kms to our signature suite. Now that I see this it doesn't look good either. Dima please revert back. I expected this type to be passed in from resolver - not hard-coded.

Reply:
The type is now a `string`, not `kms.KeyType`. I decided it will look simpler in the tests to use just constant instead of `string(kms.Ed25519Type)` (plus extra import).

Reply:
I updated constants usage. As @troyronda suggested, I added string key type constants to the KMS and re-used it for KeyType constants.

Reply:
sounds good